src/content/docs/mobile-apps/ios-widgets.mdx

```diff
@@ -20,45 +20,118 @@ import mobileappsiOSWidgetsonTodayView from 'images/mobile-apps_screenshot-full_
 
 import mobileappsiOSWidgetsAddWidgetFromChart from 'images/mobile-apps_screenshot-full_iOS-widgets-add-widget-from-chart.webp'
 
-To set up [iOS widgets](https://support.apple.com/en-us/HT207122) on your iPhone or iPad, you must be running iOS 14 or higher and have the [New Relic mobile app](install-newrelic-ios-mobile-app) installed.
+
+
+import iOSWidgetsIssueCounts from 'images/mobile-apps_screenshot-full_iOS-widgets-issue-counts.webp'
+
+import iOSWidgetsServiceLevels from 'images/mobile-apps_screenshot-full_iOS-widgets-service-levels.webp'
+
+import iOSWidgetsSyntheticMonitors from 'images/mobile-apps_screenshot-full_iOS-widgets-synthetic-monitors.webp'
+
+import iOSWidgetsNRQLQueries from 'images/mobile-apps_screenshot-full_iOS-widgets-nrql-queries.webp'
+
+
+The key to success in DevOps and site reliability engineering lies in reducing your mean time to resolution (MTTR). Widgets can help you do that, even when you're not in front of your computer.    
+
+## Before you get started with widgets [#requirements]
+
+To set up [iOS widgets](https://support.apple.com/en-us/HT207122) on your iPhone or iPad, make sure you're running iOS 14 or higher and have the [New Relic mobile app](install-newrelic-ios-mobile-app) installed.
+
+## Available iOS widgets [#available-ios-widgets]
+
+A variety of widgets are available to give you control wherever you are:
+
+<SideBySide>
+  <Side>
+    **Issue counts:** 
+
+    <img
+      style={{ maxWidth: '290px' }}
```

Comment:
The flow of this page is really awkward because of the size of the screenshots. I am sure it's more trouble than it's worth, but it might be a good time to use the side-by-side component.

Reply:
Let's think about adjusting the flow in the next round.